packages/preview/modern-szu-thesis/0.1.0/layouts/mainmatter.typ

```diff
@@ -0,0 +1,177 @@
+#import "@preview/anti-matter:0.0.2": anti-front-end

```

Comment:
`anti-matter` is [deprecated](https://github.com/tingerrr/anti-matter/issues/3). It is recommended to re-implement manually the parts that you need.